build.py

```diff
@@ -667,6 +667,16 @@ def onnxruntime_cmake_args(images, library_paths):
             else TRITON_VERSION_MAP[FLAGS.version][2],
         )
     ]
+    # If variable is set, ONNX Runtime package will be downloaded from the specified URL
+    if os.getenv("TRITON_ONNXRUNTIME_PACKAGE_URL"):
```

Comment:
I would like for this to be available as a command line option as well. We can do this one or two ways, (1) `--onnxruntime-package-url` or (2) `--extra-backend-cmake-arg=onnxruntime:TRITON_ONNXRUNTIME_PACKAGE_URL=http://example.com`.  

My reasoning here is that it should be clear from the command what behavior we are inducing in the build. If we are relying _only_ on an environment variable then it will make the build that much more opaque to the user and for us when/if we need to debug a user's issue.